InnerEye/ML/configs/histo_configs/classification/DeepSMILECrck.py

```diff
@@ -87,6 +90,22 @@ def cache_dir(self) -> Path:
             f"/tmp/innereye_cache1/{self.__class__.__name__}-{self.encoder_type}/"
         )
 
+    def setup(self) -> None:
+        if self.encoder_type == InnerEyeSSLEncoder.__name__:
+            self.downloader = CheckpointDownloader(
+                azure_config_json_path=get_workspace(),
+                run_id="updated_transforms:updated_transforms_1636471522_5473e3ff",
+                checkpoint_filename="best_checkpoint.ckpt",
+                download_dir="outputs/",
+                remote_checkpoint_dir=Path("outputs/checkpoints")
+            )
+            os.chdir(fixed_paths.repository_root_directory().parent)
+            self.downloader.download_checkpoint_if_necessary()
+
+        self.encoder = self.get_encoder()
+        self.encoder.cuda()
+        self.encoder.eval()
```

Comment:
[Suggestion] Here I'd call `super().setup()` instead.

Reply:
Not sure I get this comment - I moved the download of the checkpoint to the child class. We need different containers to download different checkpoints, it seemed confusing to have one tcga-crck checkpoint in BaseMIL.

Reply:
My bad, this would raise the `NotImplementedError` from the parent class. Nevermind :)